src/container.ts

```diff
@@ -119,6 +120,15 @@ export class DockerodeContainer implements Container {
     };
   }
 
+  public async putArchive(stream: Readable, containerPath: string): Promise<void> {
+    const respStream = await this.container.putArchive(stream, { path: containerPath });
+    await new Promise((resolve, reject) =>
+      this.container.modem.followProgress(respStream, (err: unknown, res: unknown) =>
```

Comment:
Is it necessary to follow progress here?

Reply:
I tought at first that it would be suitable to use `followProgress` here, but on second look, it seems I can also just do `await streamToArray(respStream)` instead.